public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php

```diff
@@ -606,64 +609,65 @@ public static function display_meta_boxes(
 				<?php if ( 'checkbox' == $value ) : ?>
 
 					<p>
-						<strong><?php echo $key; ?></strong>:
-						<input type="checkbox" name="<?php echo $object_name; ?>"
-							   id="<?php echo $object_name; ?>" <?php checked( get_post_meta( $post_id, $key, true ) ); ?><?php echo $readonly; ?> />
+						<strong><?php echo esc_html( $key ); ?></strong>:
```

Comment:
- [ ] Test this and below fields with filled meta data and check if rendering properly.